int32 particle_id
bool is_dynamic  # true for dynamic, false for static